### ISSUE TITLE
🐛 VoiceOver can't navigate through the page on its own

### ISSUE THREAD

**Issue by pixelasticity** [los-angeles-star/LosAngelesStar#23]
## Bug Report

**Current Behavior**
When a user enters or reloads the page while using VoiceOver, the text-to-speech program gets stuck in a loop on the stock market information.

Note: this likely affects other screen readers (e.g., JAWS, NVDA), but it hasn't been tested on them yet.

**Expected behavior/code**
VoiceOver should be able to read through the header information and move on to the body text of the page.

**Environment**
- OS: macOS 12.1 "Monterey"

**Possible Solution**
Change `display: none` on the stock ticker elements to a `visually-hidden` class.
